cpp/src/slice2cpp/Gen.h

```diff
@@ -62,7 +62,7 @@ namespace Slice
         class ForwardDeclVisitor final : public ParserVisitor
         {
         public:
-            ForwardDeclVisitor(::IceInternal::Output&);
+            ForwardDeclVisitor(::IceInternal::Output& h, ::IceInternal::Output& c);
```

Comment:
Whenever you add .cpp support, you also need dllExport.

Reply:
Also fixed!